packages/firebase_performance/example/android/gradlew

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```

Comment:
Please remove `gradlew`, `gradlew.bat`, and `gradle/wrapper/gradle-wrapper.jar`. While these files are supposed to be checked in by user projects, we cannot have them in our repos. They should be added to relevant `.gitignore` files instead.